utils/compatibility/utils.py

```diff
@@ -62,7 +62,14 @@ def latest_kube_version():
         latest = response.text.lstrip("v")
         latest = latest.split(".")
```

Comment:
this should probably use a python semver parser and ignore rc versions too.

Reply:
ack, merging these changes into the other PR to use the semver util:
https://github.com/pluralsh/console/pull/1524